proto/select.proto

```diff
@@ -130,3 +130,18 @@ message DAGRequest {
 	// It represents which columns we should output.
 	repeated uint32 output_offsets = 5;
 }
+
+enum EncodeType {
+        TypeDefault = 0;
+        TypeArrow = 1;
+}
+
+message StreamResponse {
```

Comment:
Why we need a `StreamResponse` ?

Reply:
Since some field_type of SelectResposne is redundant in Streaming API while we still need to keep `SelectResponse` in Old API.